variables.tf

```diff
@@ -49,7 +49,10 @@ variable "api_gateway_throttling_rate_limit" {
   description = "APIGateway throttling rate limit, default is -1 which does not enforce a limit"
   default     = -1
 }
-
+variable "api_gateway_timeout_milliseconds" {
+  description = "APIGateway integration request timeout (in milliseconds)"
+  default     = 6000
```

Comment:
@mb2con  Would leave this default value at 29000, that way this is an override and all existing tenants should remain as they are, see https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/api_gateway_integration#timeout_milliseconds

Reply:
changed to 29000